black_it/utils/json_pandas_checkpointing.py

```diff
@@ -174,7 +174,7 @@ def save_calibrator_state(  # pylint: disable=too-many-arguments,too-many-locals
 
     # save instantiated samplers and loss functions
     with open(checkpoint_path / "samplers_pickled.pickle", "wb") as fb:
-        pickle.dump(samplers, fb)
+        pickle.dump(scheduler, fb)
```

Comment:
should `samplers_pickle` be renamed? maybe not...

Reply:
probably yes actually, I'll take care of that